news/cookie-news.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* <news item>
+
+**Deprecated:**
+
+* <news item>
+
+**Removed:**
+
+* <news item>
+
+**Fixed:**
+
+* Repo structure modified to the new diffpy standard
```

Comment:
Maybe need to have "cookiecutter" in the news? since we will be mostly use that term when we revisit?

Reply:
"Repo structure modified to the new diffpy (cookiecutter) standard"? And we should update the instruction on here too? https://github.com/Billingegroup/cookiecutter/tree/main

Reply:
Yes, if you could make an issue, i will address it

Reply:
I created an issue in cookiecutter repo: https://github.com/Billingegroup/cookiecutter/issues/185